vue-frontend/src/components/ios-app-development/SuccessStory.vue

```diff
@@ -0,0 +1,255 @@
+    </div>
+    <!-- Mobile UI END -->
+    <!-- Desktop UI -->
+    <div
+      class="tw-hidden tw-w-full tw-h-[850px] lg:tw-h-[750px] tw-bg-[#FFF8F5] md:tw-flex tw-justify-center tw-items-center tw-relative tw-overflow-hidden before:tw-content-[''] before:tw-absolute before:tw-bottom-[0%] lg:before:tw-bottom-[-8%] before:tw-w-full before:tw-bg-[#fff] before:tw-h-[7%] before:tw-rounded-[50%] before:tw-shadow-[15px_1px_0_100px_rgba(255,255,255)] after:tw-content-[''] after:tw-absolute after:tw-top-[-2%] lg:after:tw-top-[-11%] after:tw-w-full after:tw-bg-[#fff] after:tw-h-[10%] after:tw-rounded-[50%] after:tw-shadow-[15px_1px_0_100px_rgba(255,255,255)] top-curve xll:tw-container"
+    >
+      <swiper
+        dir="rtl"
+        :slidesPerView="1"
+        :speed="2000"
+        :loop="true"
+        :autoplay="{
+          delay: 2000,
+          disableOnInteraction: false,
+        }"
+        :breakpoints="{
+          '1400': {
+            slidesPerView: 1.28,
+          },
+          '1700': {
+            slidesPerView: 1.3,
+          },
+          '2450': {
+            slidesPerView: 1,
+          },
+        }"
+        :allowTouchMove="false"
+        :spaceBetween="20"
+        :pagination="pagination"
+        :modules="modules"
+        class="swiper-container tw-basis-[116rem] lg:tw-basis-[156rem] xl:tw-basis-[175rem]"
+      >
+        <swiper-slide
+          v-for="(client, index) in clients"
+          :key="index"
+          class="tw-flex tw-justify-center tw-items-center"
+        >
+          <div
+            class="tw-ml-4 lg:tw-ml-12 tw-p-5 tw-text-white tw-flex tw-flex-row tw-items-center tw-justify-end card-contain"
+          >
+            <div
+              class="author tw-drop-shadow-md tw-rounded-[15px] tw-flex tw-flex-col tw-bg-white tw-h-[337px] tw-w-[20rem] lg:tw-w-[25rem] xl:tw-w-[30rem] tw-text-left card-rotate"
+            >
+              <div class="tw-my-auto tw-px-12">
+                <span class="tw-flex tw-flex-row tw-justify-end tw-mb-2.5">
+                  <font-awesome-icon
+                    v-for="i in 5"
+                    :key="i"
+                    class="fa-star tw-w-6 tw-h-6 footer-icon"
+                    icon="star"
+                /></span>
+                <div class="tw-flex tw-flex-col">
+                  <span
+                    class="tw-font-inter-semibold tw-text-black-core/[0.87] tw-text-xl tw-leading-[1.875rem]"
+                    >{{ client.name }}</span
+                  >
+                  <span
+                    class="tw-font-inter-regular tw-text-black-core/[0.60] tw-text-[1.125rem] tw-leading-[1.6875rem] tw-mt-[0.3rem]"
+                    >{{ client.designation }}</span
+                  >
+                </div>
+              </div>
+            </div>
+          </div>
+        </swiper-slide>
+      </swiper>
+
+      <swiper
+        :slidesPerView="1"
+        :speed="2000"
+        :loop="true"
+        :allowTouchMove="false"
```

Comment:
Figma use toch moves, have you decided with sir not to use toch moves?

Reply:
no its not required.